src/command/verify/verify.c

```diff
@@ -335,22 +437,24 @@ verifyBackupInfoFile(void)
 
             // Attempt to load the copy and report on it's status but don't keep it in memory
             const VerifyInfoFile verifyBackupInfoCopy = verifyInfoFile(
-                INFO_BACKUP_PATH_FILE_COPY_STR, false, cfgOptionStrNull(cfgOptRepoCipherPass));
+                INFO_BACKUP_PATH_FILE_COPY_STR, false, cfgOptionStrNull(cfgOptRepoCipherPass), errorList);
 
             // If the copy loaded successfully, then check the checksums
             if (verifyBackupInfoCopy.errorCode == 0)
             {
                 // If the info and info.copy checksums don't match each other than one (or both) of the files could be corrupt so
                 // log a warning but must trust main
                 if (!strEq(verifyBackupInfo.checksum, verifyBackupInfoCopy.checksum))
-                    LOG_DETAIL("backup.info.copy does not match backup.info");
+                {
+                    verifyErrorNew(errorList, logLevelDetail, strNewZ("backup.info.copy does not match backup.info"));
+                }
```

Comment:
It appears that the surrounding code doesn't use braces for 1-statement code blocks 
```suggestion
                    verifyErrorNew(errorList, logLevelDetail, strNewZ("backup.info.copy does not match backup.info"));
```